### PR TITLE
#256 Github actions publishing Storybook and NPM

### PR DESCRIPTION
**Pull request by erikmjohnson**
#256 

Added npm publishing and storybook publishing in github action.

### CODE REVIEW

File: .github/workflows/publishnpm.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+name: Publish NMP
+
+on:
+  release:
+    types: [created]
+    tags:
+      - v5.*
```
Comment (anthonyjgrove) [inline .github/workflows/publishnpm.yml]: why is this`v5` ?

File: .github/workflows/publishnpm.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+name: Publish NMP
```
Comment (anthonyjgrove) [inline .github/workflows/publishnpm.yml]: Spelling `NPM`

File: package.json [hunk 3]
```diff
@@ -61,7 +61,7 @@
     "@storybook/addons": "^5.2.1",
     "@storybook/react": "^5.2.1",
     "@storybook/storybook-deployer": "^2.8.1",
-    "autoprefixer": "^9.5.0",
+    "autoprefixer": "^9.7.1",
```
Comment (anthonyjgrove) [inline package.json]: why these updated?

File: .github/workflows/publishstorybook.yml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+      - name: Install
+        run: npm ci
+      - name: Bundle
+        run: npm run bundle --if-present
+      - name: Test
+        run: npm run test
+      - name: Deploy Storybook
+        env:
+          CI: true
+          GH_TOKEN: ${{ secrets.NOT_A_SECRET }}
```
Comment (anthonyjgrove) [inline .github/workflows/publishstorybook.yml]: can you rename the secret to be called `GH_TOKEN` not `NOT_A_SECRET`

File: package.json (hunk 3 above)
Comment (erikmjohnson) [reply package.json]: Did npm update when my shell requested it.

**Pull request merged**